static/src/javascripts/projects/commercial/modules/dfp/render-advert.ts

```diff
@@ -71,47 +70,19 @@ sizeCallbacks[adSizes.fluid.toString()] = (advert: Advert) =>
 		removeStyleFromAdIframe(advert, 'vertical-align'),
 	);
 
-/**
- * Trigger sticky scrolling for MPUs in the right-hand article column
- */
 sizeCallbacks[adSizes.mpu.toString()] = (advert: Advert): Promise<void> =>
-	fastdom.measure(() => {
-		if (advert.node.classList.contains('js-sticky-mpu')) {
-			if (advert.node.classList.contains('ad-slot--right')) {
-				stickyMpu(advert.node);
-			}
-			if (advert.node.classList.contains('ad-slot--comments')) {
-				stickyCommentsMpu(advert.node);
-			}
-		}
-		void fastdom.mutate(() => advert.updateExtraSlotClasses());
+	fastdom.mutate(() => {
+		advert.updateExtraSlotClasses();
```

Comment:
I'm not sure if this is still doing anything?

Reply:
Good point. I don't see the point of that function if we don't pass any classes to it.

It also has some very IE11 dependent logic so it could be simplified a lot.

I'll added a ticket to the backlog.